### ISSUE TITLE
buckets directory is called relative to working dir of execution

### ISSUE THREAD

**Issue by vogel** [stellar/stellar-core#1102]
Quoting @bartekn : "ok, I think I know what's happening. `stellar-core --newdb` (or `stellar-core --forcescp` not sure which one) creates `buckets` folder in the current directory. I guess it needs to be placed in a folder defined in config file but it's created in current dir."

The problem is: calling stellar-core with --newdb or --forcescp dir will make buckets directory relative to working dir, not in the directory where stellar-core lives. Then, when starting stellar-core using some kind of deamon manager (like supervisorctl), the working dir of stellar-core may be different.

--newdb is responsible for removing old buckets directory. When calling with wrong working dir, it will do nothing and leave old files in place. This can cause bugs.

Not sure what the desired behavior is. Requiring absolute path for buckets directory would be the easiest and would require only one change per instance. Using binary dir instead of working dir to resolve buckets directory could make some existing configurations not working.

**Comment by raymens** [comment]
What about using the directory where the config file is located as relative directory?

**Comment by jedmccaleb** [comment]
newdb should get the bucket dir from the stellar-core.cfg

**Comment by MonsieurNicolas** [comment]
to clarify is it that some people ran into issues because they use relative folders in the config file and it's unclear to what they are relative to?

**Comment by vogel** [comment]
Yes, that is the issue. I think it is relative to working dir but people are expecting it to be relative to stellar-core binary dir.

**Comment by graydon** [comment]
I don't think this is a bug. it has to make a choice and "relative to cwd" is as reasonable as any.
